### ISSUE TITLE
Send message error while no internet

### ISSUE THREAD

**Issue by ajaygajra** [cometchat-pro/react-native-chat-sdk#8]
https://forum.cometchat.com/t/comet-chat-failing-and-app-crashing-when-no-network-is-available/1372

**Comment by ebrake** [comment]
Could be related to https://forum.cometchat.com/t/connectionlistener-shows-connected-when-there-is-no-network-connection/1394/3 where the SDK thinks that it is connected when there is no network connection

**Comment by arunnambissan** [comment]
I am using the below code for sending messages. It’s not going inside the catch callback when there is no internet or the sendMessage function is failing.

```
CometChat.sendMessage(textMessage).then(
    message => {
        console.log("Message sent successfully:", message);
        resolve(message);
    },
    error => {
        console.log("Message sending failed with error:", error);
        reject(error)
    }
);
```

**Comment by mayur-bhandari** [comment]
This has been resolved in v2.0.9
